rpc/regen.sh

```diff
@@ -1,3 +1,8 @@
 #!/bin/sh
 
 protoc -I. api.proto --go_out=plugins=grpc:walletrpc
+
+# fix uid mapping on files created within the container
```

Comment:
is it possible to only run this from the docker script and not the generic region.sh?

Reply:
`chown` needs to be run inside the container in order to work around the docker volume file ownership issue.  I wanted to avoid having to add another script just for this -  with the below fix the command runs quietly and harmlessly.